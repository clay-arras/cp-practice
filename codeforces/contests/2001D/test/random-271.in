1
7
2 13 8 16 16 15 3
