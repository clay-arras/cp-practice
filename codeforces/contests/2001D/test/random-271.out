6
2 13 8 16 15 3
